unlock-app/src/components/creator/members/GrantKeysDrawer.tsx

```diff
@@ -252,23 +249,19 @@ const GrantKeyForm = ({ onGranted, lock }: GrantKeyFormProps) => {
             })}
           />
           {errors.recipient && (
-            <p className="text-xs -mt-4 text-[#f24c15]">
+            <p className="text-xs text-[#f24c15]">
```

Comment:
This error message is not correct. Also are you validating email addresses?

Reply:
oh! thk! 
actually, the input is 'email type', it take basic form validation, also changed with the correct error!